python/ray/data/_internal/simple_block.py

```diff
@@ -118,7 +118,7 @@ def to_block(self) -> List[T]:
         return self._items
 
     def size_bytes(self) -> int:
-        return sys.getsizeof(self._items)
+        return len(cloudpickle.dumps(self._items))
```

Comment:
This will add overhead, but unfortunately I don't know a better solution right now. Certainly it can go away once we deprecate SimpleBlock.

Reply:
Can we add a `NOTE` comment here to alert the overhead?

Reply:
Done.